### ISSUE TITLE
Ideas

### ISSUE THREAD

**Issue by ankane** [ankane/setup-opensearch#1]
Please create a new issue to discuss any ideas.

- [x] Add support `ubuntu-18.04` - waiting on https://github.com/opensearch-project/OpenSearch/issues/1591
- [x] Add support `macos-11` and `macos-10.15` - waiting on https://github.com/opensearch-project/opensearch-build/issues/38 and the Java 8 issue above
- [ ] Add support for `windows-2019` - waiting on https://github.com/opensearch-project/opensearch-build/issues/33
- [x] Add support for custom port - either `port` option ([port branch](https://github.com/ankane/setup-opensearch/compare/port)) or parse `http.port` from `config` option (merged)
